packages/grafana-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -64,6 +64,7 @@ export const Dropdown = React.memo(({ children, overlay, placement, offset, onVi
     onOpenChange: handleOpenChange,
     middleware,
     whileElementsMounted: autoUpdate,
+    strategy: 'fixed',
```

Comment:
Inside the split view the dropdowns aren't rendered in the correct position otherwise.

Reply:
don't they use a portal?

Reply:
The issue was that the portal was inside the left pane only so the positioning wasn't right. When the portal root is in the top level this is not needed anymore so removing it.